From 3e7a0f39e5acf4e9da96cb075050e51e24e81403 Mon Sep 17 00:00:00 2001
From: rtogni <rtogni@b3059339-0415-0410-9bf9-f77b7e298cf2>
Date: Sun, 7 Aug 2016 20:44:47 +0000
Subject: [PATCH] Fix vo_png with recent ffmpeg

avcodec_open2() now requires timebase to be always set, even for png images.
The patch sets it to 1/1, since pictures do not have a framerate.
---
 libvo/vo_png.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libvo/vo_png.c b/libvo/vo_png.c
index 38a4097..5696a1f 100644
--- a/libvo/vo_png.c
+++ b/libvo/vo_png.c
@@ -136,6 +136,8 @@ config(uint32_t width, uint32_t height, uint32_t d_width, uint32_t d_height, uin
         avctx->pix_fmt = imgfmt2pixfmt(format);
         avctx->width = width;
         avctx->height = height;
+        avctx->time_base.num = 1;
+        avctx->time_base.den = 1;
         if (avcodec_open2(avctx, avcodec_find_encoder(AV_CODEC_ID_PNG), NULL) < 0) {
             uninit();
             return -1;
